homeassistant/components/sensor/isy994.py

```diff
@@ -309,3 +312,47 @@ def unit_of_measurement(self) -> str:
             return self.hass.config.units.temperature_unit
         else:
             return raw_units
+
+
+class ISYWeatherDevice(isy.ISYDevice):
+    """Representation of an ISY994 weather device."""
+
+    _domain = 'sensor'
+
+    def __init__(self, node) -> None:
+        """Initialize the ISY994 weather device."""
+        isy.ISYDevice.__init__(self, node)
+
+    @property
+    def unique_id(self) -> str:
+        """Return the unique identifier for the node."""
+        return self._node.name
+
+    @property
+    def raw_units(self) -> str:
+        """Return the raw unit of measurement."""
+        if self._node.uom == 'F':
+            return TEMP_FAHRENHEIT
+        if self._node.uom == 'C':
+            return TEMP_CELSIUS
+        return self._node.uom
+
+    @property
+    def state(self) -> object:
+        """Return the value of the node."""
+        # pylint: disable=protected-access
+        val = self._node.status._val
+        raw_units = self._node.uom
```

Comment:
No I'm definitely sure. It comes in as a list of units because `uom` could be unit of measure or enumerated device states (on, off, open, closed, etc.). If the device has one element in the `uom` list then it's a unit of measurement.